tools/api_check/lib/apicheck.dart

```diff
@@ -35,8 +35,9 @@ List<String> getDartClassFields({
   final RegExp fieldExp = RegExp(r'_k(\w*)Index');
   final List<String> fields = <String>[];
   for (final CompilationUnitMember unitMember in result.unit.declarations) {
-    if ((unitMember is ClassDeclaration || unitMember is EnumDeclaration) && unitMember.name.lexeme == className) {
-      for (final ClassMember classMember in unitMember.members) {
+    final NodeList<ClassMember> members = _getMembers(className, unitMember);
```

Comment:
if you don't like my other suggestion, you'll have to type this as nullable:

```suggestion
   final NodeList<ClassMember>? members = _getMembers(className, unitMember);
```

Reply:
oh, it was nullable but I added the if as a precaution. Uhhhhhh. Thanks for your patience.

Reply:
I'm not that bad, but half my changes I see this and think I got it right: 
![image](https://user-images.githubusercontent.com/351125/230676968-8f535fb3-3985-4425-9315-3542d8da23ba.png)